### DISCUSSION TITLE
not able to add facebook chat plugin

### DISCUSSION THREAD

**Discussion by chris110408** [arthelokyo/astrowind#382] [Q&A] [answered]
try to add the facebook chat plugin however, it does not work. 

I put the chat plugin script into the basicscript file
and tried to add the  HTML in both layout and  pageLayout
``` 
   <div id="fb-root"></div>

    <!-- Your Chat Plugin code -->
    <div id="fb-customer-chat" class="fb-customerchat"></div>

**Comment by prototypa** [comment, marked as answer]
Hi @chris110408 

To add the FB Chat Plugin I recommend that you create a new component with all the code suggested by FB. Then you go to `Layout.astro` and import it and include it under the `<BasicScripts />` component: https://github.com/onwidget/astrowind/blob/main/src/layouts/Layout.astro#L47

Anything you let me know. I have moved this issue to Discussion since it is not a problem with **AstroWind**.
